vc_zoom/indico_vc_zoom/plugin.py

```diff
@@ -0,0 +1,464 @@
+                                 update_zoom_meeting)
+
+
+class PluginSettingsForm(VCPluginSettingsFormBase):
+    _fieldsets = [
+        ('API Credentials', ['api_key', 'api_secret', 'webhook_token']),
+        ('Zoom Account', ['email_domains', 'assistant_id', 'allow_webinars']),
+        ('Room Settings', ['mute_audio', 'mute_host_video', 'mute_participant_video', 'join_before_host',
+                           'waiting_room']),
+        ('Notifications', ['zoom_phone_link', 'creation_email_footer', 'send_host_url'])
+    ]
+
+    api_key = StringField(_('API Key'), [DataRequired()])
+
+    api_secret = IndicoPasswordField(_('API Secret'), [DataRequired()], toggle=True)
+
+    webhook_token = IndicoPasswordField(_('Webhook Token'), toggle=True,
+                                        description=_("Specify Zoom's webhook token if you want live updates"))
+
+    email_domains = StringField(_('E-mail domains'), [DataRequired()],
+                                description=_("Comma-separated list of e-mail domains which can use the Zoom API."))
+
+    assistant_id = StringField(_('Assistant Zoom ID'), [DataRequired()],
+                               description=_('Account to be used as owner of all rooms. It will get "assistant" '
+                                             'privileges on all accounts for which it books rooms'))
+
+    allow_webinars = BooleanField(_('Allow Webinars'),
+                                  widget=SwitchWidget(),
+                                  description=_('Allow webinars to be created through Indico'))
+
+    mute_audio = BooleanField(_('Mute audio'),
+                              widget=SwitchWidget(),
+                              description=_('Participants will join the VC room muted by default '))
+
+    mute_host_video = BooleanField(_('Mute video (host)'),
+                                   widget=SwitchWidget(),
+                                   description=_('The host will join the VC room with video disabled'))
+
+    mute_participant_video = BooleanField(_('Mute video (participants)'),
+                                          widget=SwitchWidget(),
+                                          description=_('Participants will join the VC room with video disabled'))
+
+    join_before_host = BooleanField(_('Join Before Host'),
+                                    widget=SwitchWidget(),
+                                    description=_('Allow participants to join the meeting before the host starts the '
+                                                  'meeting. Only used for scheduled or recurring meetings.'))
+
+    waiting_room = BooleanField(_('Waiting room'),
+                                widget=SwitchWidget(),
+                                description=_('Participants may be kept in a waiting room by the host'))
+
+    zoom_phone_link = URLField(_('ZoomVoice phone number'),
```

Comment:
"ZoomVoice" sounds a lot like a leftover from copying the field about "VidyoVoice" in the Vidyo plugin

Reply:
Yeah, I forgot to check whether having something like this is still useful.